refocus-bdk-server.js

```diff
@@ -1067,7 +1068,8 @@ module.exports = (config) => {
       const events = {
         log: msg,
         roomId: room,
-        actionType: type
+        actionType: type,
+        botId: botName
```

Comment:
same question here

Reply:
Wonder if that is not a problem for a non-imc expert... is this happening in each bot or just in the BDK?